maps/sccv_horizon/sccv_horizon-1_deck_1.dmm

```diff
@@ -12618,7 +12618,7 @@
 	frequency = 1442;
 	input_tag = "port_prop_in";
 	output_tag = "port_prop_out";
-	sensors = list("port_prop_sensor"="Port Propulsion Sensor")
+	sensors = list("port_prop_sensor"="Port    Propulsion    Sensor")
```

Comment:
intended?

Reply:
i have no idea why this happened i can only assume it was some wacky gitkraken issue

Reply:
fixed